refellips/objectiveSE.py

```diff
@@ -178,9 +178,13 @@ def residuals(self, pvals=None):
 
         wavelength, aoi, psi_d, delta_d = self.data.data
         wavelength_aoi = np.c_[wavelength, aoi]
-
         psi, delta = self.model(wavelength_aoi)
-        return np.r_[psi - psi_d, delta - delta_d]
+        delta_err = 2 * np.rad2deg(
```

Comment:
If you want to do this calculation, we should probably do it in `ReflectModelSE`.

Reply:
Actually, scrub that. What we're working out here is the residual for delta, which you're effectively changing to:
```
2 * asin(sin(delta/2) - sin(delta_d/2))
```
? Why is the residual for `delta` worked out this way? Do you have a reference?

Reply:
Delta is measured in degrees; as such, the difference between ∆=359 and ∆=1 should be 2˚. In the old implementation, it was 358˚.

In this method, the delta values are transformed such that 180˚=1, and 0/360˚=0. The residual is calculated, then rescaled such that the magnitude is the same as a simple difference.

I haven't managed to find a reference for the 'correct' way to calculate the residual, but I can demonstrate why the new method makes more sense than the old method by way of example (& comparison to WVASE).

I haven't convinced myself that this is the best way of calculating the residual - will do more searching.

Reply:
Some sort of circular modulus

Reply:
<img width="588" alt="Screenshot 2024-10-04 at 8 00 40 pm" src="https://github.com/user-attachments/assets/341ef191-08c3-4c62-893e-e8addb8d81ce">

Seems like: 
delta_error = (delta - delta_d + 180) % 360 - 180

produces the desired effect.

Reply:
For reference, the problem with the old approach is:
<img width="659" alt="Screenshot 2024-10-04 at 9 11 23 pm" src="https://github.com/user-attachments/assets/24521e61-d193-4f71-a148-dc32bfba6553">